float64[12] x0
float64 t0
convex_mpc_controller/init_pos init_stance_pos
float64[2] vel_des
float64 yaw_rate_des
